onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -115,6 +126,17 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
   if ((global_context_.use_compiled_network == true && import_blob_status == false) || vpu_status == false) {
     if(!openvino_ep::backend_utils::UseCompiledNetwork()) {
       ie_cnn_network_ = CreateCNNNetwork(model_proto, global_context_, subgraph_context_, const_outputs_map_);
+
```

Comment:
@tsocha Please add a check #if defined(OPENVINO_2021_1) before start of your code and close with **#endif**

Reply:
Done